pkg/libvmi/cloudinit.go

```diff
@@ -57,6 +59,8 @@ func WithCloudInitNoCloudNetworkData(data string) Option {
 
 		volume := getVolume(vmi, cloudInitDiskName)
 		volume.CloudInitNoCloud.NetworkData = data
+		volume.CloudInitNoCloud.NetworkDataBase64 = ""
```

Comment:
this line fixes an old bug, consider putting it in its own commit.

Reply:
Done